tests/api/collections.rs

```diff
@@ -92,14 +93,15 @@ async fn test_create_and_get_collection_with_empty_title() -> Result<(), Error>
     assert_eq!(bookmarked["title"], "CSS: Cascading Style Sheets");
     assert_eq!(bookmarked["url"], "/en-US/docs/Web/CSS");
 
+    drop(stubr);
```

Comment:
explicitly droping stubr seems to fix the flakiness, we can try and remove it again. If it really works we can look more into upstream stubr.